app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormBranch.test.tsx

```diff
@@ -109,7 +97,7 @@ describe("FormBranch", () => {
     const featureIdx = 1;
     const onFeatureConfigChange = jest.fn();
     render(<SubjectBranch {...{ onFeatureConfigChange }} />);
-    fireEvent.change(screen.getByTestId("feature-config-select"), {
+    fireEvent.change(screen.getByTestId("referenceBranch.featureConfig"), {
```

Comment:
Q: Why are these "referenceBranch.featureConfig"? before they were kebab case which makes me wonder if these are (a) inconsistent, (b) references and not IDs, (c) variables, (d) other.

Although a Ctrl+F of this PR did turn up this, so maybe we weren't always using kebab casing 100% previously `_feature_select_locator = (By.CSS_SELECTOR, "#referenceBranch-featureConfig")`. Although one is `referenceBranch.featureConfig` (period) and the other is `referenceBranch-featureConfig` (hyphen).

Reply:
We have kind of a mess of casing since the casing from the server is snake, we're mostly using camel on the client, and then kebab in some tests/IDs. The test ID here changed because I needed to add `formControlAttrs` and it sets `test-id` based on the `fieldName`, which is `${prefix}.${name}` or in this case, `referenceBranch.featureConfig`. This has to do with some branch form nesting and what was easiest to do there when we were first making the branches forms. Could maybe use some cleanup but may also not be worth it at least in this particular instance, IMO, but we could probably go through and see what we _can_ make consistent without a high level of effort refactoring. 🤔